src/streamlink/plugins/zattoo.py

```diff
@@ -175,21 +184,31 @@ def _hello(self):
         params = {
             'client_app_token': app_token,
             'uuid': __uuid,
-            'lang': 'en',
-            'format': 'json'
         }
-        res = self.session.http.post(hello_url, headers=self.headers, data=params)
+
+        if self.base_url == 'https://www.quantum-tv.com':
+            params['app_version'] = '3.2028.3'
+        else:
+            params['lang'] = 'en'
+            params['format'] = 'json'
+
+        res = self.session.http.post(hello_url, headers=self.headers, data=params, cookies=self.session.http.cookies)
```

Comment:
```
cookies=self.session.http.cookies
```

are you sure this is needed, since this is already a session.

Reply:
no it isn't needed, was a leftover from testing